lochness/utils/source_check.py

```diff
@@ -269,6 +272,7 @@ def check_source(Lochness: 'lochness') -> None:
         db_string = 'RPMS'
         mediaflux_df = collect_mediaflux_files_info(Lochness)
         all_df = check_file_path_df(mediaflux_df, subject_id_list)
+        all_df = all_df[all_df['site'].str.startswith('Prescient')]
```

Comment:
Are we reducing the `all_df` for speed purpose down the line?

Reply:
There are other directories in mediaflux, which are not being used in the Prescient study. These directories are being skipped.